aspnetcore/client-side/spa/blazor/index.md

```diff
@@ -48,7 +48,7 @@ Blazor supports core facilities required by most apps, including:
 * Templating
 * Cascading values
 
-To reduce the size of the downloaded app unused code stripped out of the app when it's published by the [Intermediate Language (IL) Linker](xref:host-and-deploy/razor-components/configure-linker).
+To reduce the size of the downloaded app unused code stripped out of the app when it's published by the [Intermediate Language (IL) Linker](xref:host-and-deploy/razor-components-blazor/configure-linker).
```

Comment:
Doesn't that need a comma?
To reduce the size of the downloaded app,  unused code **is** stripped out of the app when it's published by the

Reply:
oooooooooooooooooooops a daisy!